frame/support/src/traits/tokens/currency/lockable.rs

```diff
@@ -99,6 +99,9 @@ pub trait VestingSchedule<AccountId> {
 
 	/// Remove a vesting schedule for a given account.
 	///
+	/// Parameter `schedule_index` is only applicable for implementations that
+	/// support multiple schedules.
+	///
 	/// NOTE: This doesn't alter the free balance of the account.
-	fn remove_vesting_schedule(who: &AccountId);
+	fn remove_vesting_schedule(who: &AccountId, schedule_index: Option<u32>) -> DispatchResult;
```

Comment:
Trying not minimize the breaking change for this trait by making the new param an option, but not sure how helpful that is

Reply:
My rule would be to only make it an `Option` if the function can do something useful with `None`. It looks like it just errs out in that case, which is not all that useful; in that case, better to just require the index.

Reply:
8abf51d